_docs/_latest/setup/upgrade.md

```diff
@@ -398,6 +398,46 @@ Example command: `gcloud compute instances reset forseti-server-vm-70ce82f --zon
 1. Configuration file `forseti_conf_server.yaml` updates:  
 Forseti is updated to be usable on a non organization resource.
 
+{% capture upgrading_2_4_0_to_2_5_0 %}
+
+Starting v2.5,Forseti Inventory Crawler will be integrated with the Cloud 
+Asset Inventory (CAI) service by default if the user is deploying Forseti. CAI
+feature is supported only if the level is `organization`.
+
+1. Users upgrading to v2.5 from v2.4 can enable CAI by creating a new bucket 
+which will be used for CAI exports and by providing it's path in
+`forseti_conf_server.yaml.in` file.
```

Comment:
Not the `.in` file.  It's just the generated `forseti_conf_server.yaml`

Reply:
Done